trino-connector/src/main/java/com/datastrato/gravitino/trino/connector/catalog/CatalogConnectorManager.java

```diff
@@ -174,15 +174,19 @@ public void loadCatalogs(GravitinoMetaLake metalake) {
                 if (!catalogConnectors.containsKey(catalogName)) {
                   Catalog catalog = metalake.loadCatalog(nameIdentifier);
 
-                  GravitinoCatalog gravitinoCatalog = new GravitinoCatalog(catalog);
-                  CatalogConnectorContext catalogConnectorContext =
-                      catalogConnectorFactory.loadCatalogConnector(
-                          nameIdentifier, metalake, gravitinoCatalog);
-
-                  catalogConnectors.put(catalogName, catalogConnectorContext);
-                  catalogInjector.injectCatalogConnector(catalogName);
-                  LOG.info(
-                      "Load catalog {} in metalake {} successfully.", catalogName, metalake.name());
+                  if (catalog.type() == Catalog.Type.RELATIONAL) {
```

Comment:
So we would not support fileset and stream catalog any more?

Reply:
Yes